app/views/warehouse_reports/outcomes/shared/_form.haml

```diff
@@ -1,24 +1,38 @@
 = simple_form_for @filter, as: :filter, url: report_index_path, method: :get do |f|
+  = debug @filter
   - content_for :filters_col_full do
-    = f.input :project_ids, collection: @available_projects, input_html: {class: :select2, multiple: true, data: {placeholder: 'All'}}, include_blank: false, label: 'Projects'
+    .row
+      .col
+        = f.input :data_source_ids, collection: @filter.data_source_options_for_select(user: current_user), placeholder: 'All', as: :select_two, input_html: { multiple: true }, include_blank: false, label: 'Data Sources'
+      .col
+        = f.input :organization_ids, collection: @filter.organization_options_for_select(user: current_user), placeholder: 'All', as: :select_two, input_html: { multiple: true }, include_blank: false, label: 'Organizations'
+      - unless GrdaWarehouse::Config.get(:multi_coc_installation)
```

Comment:
@awisema , I'm not sure what's up with organization_ids, but I can't get them to stick in the form.  The filter reports them all as having an id of 0. Can you take a look?

Reply:
Needed to be `grouped_select` -- should we add a `grouped_select_two`?

Reply:
we should